perception/autoware_tensorrt_bevdet/src/data.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "autoware/tensorrt_bevdet/data.hpp"
+
+#include "autoware/tensorrt_bevdet/cpu_jpegdecoder.hpp"
+
+#include <stdlib.h>
+
+#include <chrono>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+
+using std::chrono::duration;
+using std::chrono::high_resolution_clock;
+
+camParams::camParams(const YAML::Node & config, int n, std::vector<std::string> & cams_name)
+: N_img(n)
+{
+  if (static_cast<size_t>(n) != cams_name.size()) {
```

Comment:
The C-style cast `static_cast<size_t>(n)` should be replaced with a C++-style cast or brace initialization.  For example, `size_t{n}` is a more modern and safer approach.  This is recommended to improve code clarity and consistency.